README.md

```diff
@@ -31,13 +39,61 @@ export JWT_ISSUER=http://localhost:8082/
 export JWT_AUDIENCE=http://localhost:8082/
 export JWT_REALM=jehlomat_local_realm
 export SUPER_ADMIN_EMAIL=jehlomat@cesko.digital
-export MAILJET_PUBLIC_KEY=public-key-for-mailjet
-export MAILJET_PRIVATE_KEY=private-key-for-mailjet
+export MAILJET_PUBLIC_KEY=cbb81504a06f2fd735db577f09666b7f
+export MAILJET_PRIVATE_KEY=fdd479b47c4af6e4d46bb07bf889432d
 ```
-6) Run application
+6) cd service-jehlomat 
+7) ../gradlew shadowJar 
+8) java -jar build/libs/service-jehlomat-all.jar -config=./src/test/resources/application.conf
+
+## Run hybrid backend application with Docker
+
+> This approach is useful mostly for FE development.
+By hybrid means, the BE service will be run locally but the database will be run via docker, so it will not be necessary to install the DB and run it 1. locally.
```

Comment:
line break before please, the markdown thinks it is a part of the prev quote.

Reply:
Please see: https://github.com/cesko-digital/jehlomat/pull/163/commits/e2e2199f31ec79331afafc74a63ccbb9afac5a2d
if this matches your comment